ev3dev2/sound.py

```diff
@@ -370,10 +388,10 @@ def get_volume(self, channel=None):
         if channel is None:
             channel = self._get_channel()
 
-        out = check_output(['amixer', 'get', channel]).decode()
-        m = re.search(r'\[(?P<volume>\d+)%\]', out)
+        out = os.popen(['/usr/bin/amixer', 'get', channel]).read()
```

Comment:
this is supported on Micropython as well?

Reply:
yep, I tested every Sound feature I could find and they all work with python3 and micropython